v23.1/fips-compliance.md

```diff
@@ -0,0 +1,355 @@
+---
+title: Install a FIPS-ready CockroachDB Runtime
+summary: Learn how to install and run CockroachDB in an environment subject to FIPS 140-2.
+toc: true
+docs_area: deploy
+---
+
+## Overview of FIPS-ready CockroachDB
+
+[Federal Information Processing Standards (FIPS) 140-2](https://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.140-2.pdf) is a mandatory standard that is used to approve cryptographic models. The goal of FIPS 140-2 is to provide measurable security guidelines for handling of and access to sensitive but unclassified (SBU) information. The FIPS 140-2 standard is applicable to all federal agencies that use cryptographic-based security systems to protect sensitive information in computer and telecommunication systems (including voice systems) as defined in Section 5131 of the Information Technology Management Reform Act of 1996, Public Law 104-106 and the Federal Information Security Management Act of 2002, Public Law 107-347. U.S. and Canadian governments, as well as organizations working with them, may be subject to FIPS 140-2 requirements.
+
+The Cryptographic Module Validation Program (CMVP) validates cryptographic modules to FIPS 140-2 and other cryptography-based standards. When a cryptographic module or library has a FIPS 140-2 certificate, it has been tested and formally validated under the CMVP as meeting the requirements for FIPS 140-2.
+
+FIPS-ready CockroachDB binaries and Docker images are available for CockroachDB v23.1.0 and above. CockroachDB FIPS-ready runtimes run only on Intel 64-bit Linux systems.
+
+FIPS-ready binaries are built using [Red Hat's Go FIPS with OpenSSL toolchain](https://github.com/golang-fips/go), which contains the necessary modifications for the Go crypto library to use an external cryptographic library in a manner compatible with FIPS 140-2. FIPS-ready CockroachDB delegates cryptographic operations to the host operating system's OpenSSL libraries and commands, rather than Go's cryptographic libraries. When the installed OpenSSL has a FIPS 140-2 certificate and [FIPS mode is enabled in the Linux kernel](#enable-fips-mode-in-the-linux-kernel), the CockroachDB runtime is suitable for workloads that are subject to FIPS 140-2 requirements.
+
+For details about cryptographic algorithms and key lengths used by CockroachDB. refer to [Details About Cryptographic Algorithms](#details-about-cryptographic-algorithms).
+
+### FIPS-ready features
+
+When you use a FIPS-ready CockroachDB runtime and each cluster node's OpenSSL is FIPS-validated and configured correctly, Cockroach Labs has tested the following features to ensure that their cryptographic operations meet the requirements of FIPS 140-2:
+
+- [Encryption At Rest](security-reference/encryption.html#encryption-keys-used-by-cockroachdb-self-hosted-clusters)
+- [Encrypted Backups](take-and-restore-encrypted-backups.html)
```

Comment:
Link text can use title or sentence case depending on the context: https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#capitalization-rules Since I am declining to add periods, I'll keep these as title case.